src/main/java/com/users/validation/AdultValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.users.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import java.time.*;
+
+public class AdultValidator implements ConstraintValidator<Adult, Instant> {
+    @Override
+    public void initialize(Adult constraintAnnotation) {
+    }
+
+    @Override
+    public boolean isValid(Instant dateOfBirth, ConstraintValidatorContext context) {
```

Comment:
ai nevoie de parametrul context?

Reply:
isValid in  ConstraintValidator are metoda abstracta de genul asta -  boolean isValid(T var1, ConstraintValidatorContext var2); Cum clasa mea AdultValidator implementeaza ConstraintValdiator trebuie sa am aceiasi parametrii ca in superclasa pentru metoda asta.